Add Jenkinsfile for Pipeline for caasp-services

SlickNik opened the pull request:
Add a new Jenkinsfile which defines the pipeline for the
caasp-services repository. Initially this pipeline is identical to the
pipelines of the other CaaSP projects.

at1012 commented:
LGTM. This would make it easier to install kube-dns in the caasp cluster now. 

For later versions of caasp, we should probably use this PR [https://github.com/kubic-project/salt/compare/master...at1012:at-kubedns](url) with the images replaced with the packages that we created.

SlickNik commented:
Thanks @at1012 👍

SlickNik committed:
Add Jenkinsfile for Pipeline for caasp-services

Add a new Jenkinsfile which defines the pipeline for the
caasp-services repository. Initially this pipeline is identical to the
pipelines of the other CaaSP projects.

kiall reviewed (commented):
+1  - I just updated the Jenkins config for the repo move, and this was picked up + is running correctly.
